vagrant/lib/forklift/box_distributor.rb

```diff
@@ -93,6 +93,7 @@ def define_vm(config, box = {})
         configure_rackspace(machine, box)
         configure_openstack_provider(machine, box)
         configure_google_provider(machine, box)
+        configure_docker_provider(machine, box)
```

Comment:
Given the whole bunch, perhaps it gets to a poin where we want to introduce `configure_providers` and then call all of these providers? Probably makes sense to still keep all the separate methods, but this method is getting very long.

Reply:
Good idea!

Reply:
Done!

(Correctly I think, I have minimal ruby knowledge.)